src/_data/team.yml

```diff
@@ -652,3 +652,15 @@
     <p>Apart from coding, he enjoys read books, practice sports, trip around the world, and, above all, spend time whith his friends.</p>
   github: glippi
   twitter: gabriele_lippi
+
+- id: jorge_viana
+  name: Jorge Viana
+  image: /assets/custom/img/team/jorge_viana.jpg
+  role: Apprentice
+  fullDescription:
+    <p>Jorge had his first computer at around the age of 10, a Sinclair ZX Spectrum with 48 KB of memory – one never forgets that.</p>
+    <p>At first he tought that personal computers were made to play games, but fortunately the computer came with a manual to a programming language (BASIC) with simple and fun examples, and this changed his live forever.</p>
+    <p>He tried to learn as much as he could on his own, and later when he arrived at high school he signed up for some classes with computer related topics.</p>
+    <p>After finishing high school he had to put is passion on hold for a while and he worked for many years in many different things until he had the opportunity to go to the university to study computer science.</p>
```

Comment:
I think there is an h missing in
`he had to put is passion on hold`
vs
`he had to put his passion on hold`

Reply:
Good catch, thank you. Fixed it!